src/odemis/acq/test/millmng_test.py

```diff
@@ -405,11 +405,15 @@ def test_milling_settings_from_yaml(self):
         """
         Test mill features function using the milling settings from a YAML file
         """
+        # Milling does not start from an unknown stage position
+        # Set the stage to loading position
+        cryoSwitchSamplePosition(LOADING).result()
+
         millings = load_config(MILLING_CONFIG_PATH + "/milling-series-test-working-config.mill.yaml")
         logging.debug(millings)
         f = mill_features(millings, self.sites, self.feature_post_status, self.acq_streams, self.ion_beam,
                           self.sed, self.stage, self.aligner)
-
+        time.sleep(0.1)
```

Comment:
I would say that you could increase a bit more, to at least 1s, to make sure it's really running.

Reply:
I understand where you are coming from, but it also feels like we are testing the wrong thing if 1s is good, but 0.1s is not good. Is there another way we can test whether the milling has actually started, for instance can we check that a certain setting changed? In addition to checking `f.running()`.

Reply:
The problem is that in the simulator, we use `FakeOrsayMilling`. This class is very simple. It essentially just does a `sleep()`. Hence, there is not much else to check.